src/postgres-util/src/lib.rs

```diff
@@ -188,3 +190,8 @@ pub fn make_tls(config: &tokio_postgres::Config) -> Result<MakeTlsConnector, Pos
 
     Ok(tls_connector)
 }
+
+/// Object names surrounded in double quotes are not double-quoted in the system catalog.
+pub(crate) fn sanitize_name_for_catalog_matching(name: &str) -> &str {
+    name.trim_matches('"')
```

Comment:
What if the name of the object is a single double quote? trim_matches removes all, not just the first.

Reply:
Oh sorry--you caught a bad version of this; it's fixed.